zerver/views/invite.py

```diff
@@ -54,6 +53,7 @@ def invite_users_backend(
         default=PreregistrationUser.INVITE_AS["MEMBER"],
     ),
     stream_ids: List[int] = REQ(json_validator=check_list(check_int)),
+    subscribe_to_default_streams: bool = REQ(json_validator=check_bool, default=True),
```

Comment:
Thinking about it more, I kinda agree that `False` can be a better default. Updated the PR for now. Can change it again if feedback from others suggests otherwise.

Do we want also want to change the DB level default for this then?

Reply:
I think `True` might be fine as a database-level default, but I think we need a `RunPython` migration step to make sure that it's set correctly for preexisting invitation objects -- presumably any that had `stream_ids` set should have a value of `True`. 

While you're it it, we might want to do a multi-step migration where we first add a `null=True` field, then backfill it, and then make it non-null; these can be big tables.